runtime/src/utils.rs

```diff
@@ -148,7 +148,7 @@ macro_rules! hours {
 #[macro_export]
 macro_rules! days {
     ($a:expr) => {{
-        hours!(24) * $a
+        ((24 * 60 * 60 * 1000) / ExpectedBlockTime::get()) as u32 * $a
```

Comment:
mathematically it looks the same to me, is this implementation more accurate (due to integer maths?)

Reply:
this was in order to silence clippy, I will revisit